src/vs/workbench/contrib/notebook/browser/viewParts/notebookKernelQuickPickStrategy.ts

```diff
@@ -727,6 +732,18 @@ export class KernelPickerMRUStrategy extends KernelPickerStrategyBase {
 					};
 				}));
 
+				const sourceActionCommands = this._notebookKernelService.getSourceActions(notebook, contextKeyService);
+				sourceActionCommands.forEach(sourceAction => {
```

Comment:
for const of is generally preferred as debugging forEach needs a breakpoint in the callback